### ISSUE TITLE
[bug]: The selected indicator for radio groups is not centered

### ISSUE THREAD

**Issue by AlexanderHott** [shadcn-ui/ui#3621]
### Describe the bug

The filled circle that appears in the center of a radio check is not centered in the outer circle.

### Affected component/components

Radio Group

### How to reproduce

Present on documentation.

At different zoom levels, the circle is offset in different places
![image](https://github.com/shadcn-ui/ui/assets/66271487/e41ac91e-6088-4d4d-8128-3cad8e14d0ac)
![image](https://github.com/shadcn-ui/ui/assets/66271487/4518f825-9bda-402d-bae3-8d24f63c6ac2)
![image](https://github.com/shadcn-ui/ui/assets/66271487/b0d00d6e-a0d2-4e35-8553-8e4206544a9a)

### Codesandbox/StackBlitz link

https://ui.shadcn.com/docs/components/radio-group

### System Info

```bash
Both Brave and Firefox have this issue.
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by AlexanderHott** [comment]
This could be an issue with the lucide icons, but I'm not sure.

**Comment by voinik** [comment]
Yeah same issue here. On Arc/Chrome it works fine, but on my iPhone it looks off center. Sometimes it's centered, but most of the time it is not.

**Comment by i0rph** [comment]
I found that this bug occurs when you set the display scale above 100%. Both Chromium and Firefox affected by this setting.
We gotta wait for Microsoft or browsers to fix this.

**Comment by Lexachoc** [comment]
I can confirm that when the page is zoomed in by more than 100%, the indicator is incorrectly centred in Chrome, Edge, and even Firefox!

I tried using the PR https://github.com/shadcn-ui/ui/pull/2758, but unfortunately it is still not centred with different page zoom sizes.

**Comment by ehsanelgendi** [comment]
same issue here

**Comment by phev8** [comment]
As a workaround, I removed the Circle icon and style the Indicator directly:
```
<RadioGroupPrimitive.Indicator className="flex items-center justify-center aspect-square size-2 mx-auto bg-primary rounded-full"/>
```

Does anybody know, what's the reason for using an SVG icon (lucide icon) for the circle instead of simply styling the radix primitive?

**Comment by myieye** [comment]
#2758 didn't solve this for me either, it just changes which zoom-levels break the centering.

I'm using Svelte, but my solution translated into react was to just drop the `RadioGroupPrimitive.Indicator` wrapper, blow-up the circle to the same size as the ring and then shrink it down using scale. I.e. `<Circle className="text-current size-full scale-75 origin-center" />`.

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)

**Comment by rmontagnin** [comment]
This is still present
